arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/precompileSubsection/ModexpSubsection.java

```diff
@@ -75,6 +76,7 @@ public ModexpSubsection(final Hub hub, final CallSection callSection) {
             >= 0) {
```

Comment:
I don't remember the precise semantics of `BigInteger#compareTo` but it looks like the logic here is saying that "if either `bbs`, `ebs`, `mbs` is `≥ 512` then the `MODEXP` operation is rejected by Linea".

But the real condition is "if either `bbs`, `ebs`, `mbs` is `> 512` then the `MODEXP` operation is rejected by Linea".

> Is my interpretation correct ? If so we must replace all `>=` by `>`.

Reply:
I think you're absolutely right. I'm making the change. Very good catch.